components/eamxx/src/physics/ml_correction/eamxx_ml_correction_process_interface.cpp

```diff
@@ -81,6 +84,16 @@ void MLCorrection::initialize_impl(const RunType /* run_type */) {
   ML_model_uv = py_correction.attr("get_ML_model")(m_ML_model_path_uv);
   ML_model_sfc_fluxes = py_correction.attr("get_ML_model")(m_ML_model_path_sfc_fluxes);
   ekat::enable_fpes(fpe_mask);
+
+  // Enforce bounds on quantities adjusted by ML using Field Property Checks
+  using LowerBound = FieldLowerBoundCheck;
+  add_postcondition_check<LowerBound>(get_field_out("precip_liq_surf_mass"),m_grid,0,true);
+  add_postcondition_check<LowerBound>(get_field_out("precip_ice_surf_mass"),m_grid,0,true);
+  // Add sanity property check for other variables.
+  using Interval = FieldWithinIntervalCheck;
+  add_postcondition_check<Interval>(get_field_out("qv"),m_grid,0,0.2,true);
+  add_postcondition_check<Interval>(get_field_out("T_mid"),m_grid,100.0,500.0,false);
+  add_postcondition_check<Interval>(get_field_out("horiz_winds"),m_grid,-400.0,400.0,false);
```

Comment:
I think it's great to have bound checks, but should this be done from within ML prediction? There's threshold limits on the predicted tendencies, but I think the issue here is that the actual field can go crazy?